.husky/pre-commit

```diff
@@ -0,0 +1,3 @@
+npm run format
+npm run lint
+npm run build
```

Comment:
We use yarn, wouldn't this be yarn instead of npm?

Reply:
absolutely can be , changed in the next commit